src/builtin_types.md

```diff
@@ -69,7 +69,7 @@ enum Result<t, e> {
 }
 ```
 
-The type of Grain results (i.e. `Ok(1)` or `Err("Something went wrong")`). The `t` and `e` are the types of the value and error, respectively. This is useful for functions that might fail, and is the type of the `Result` module.
+The type of Grain results (i.e. `Ok(1)` or `Err("Something went wrong")`). The `t` and `e` are the types of the value and error, respectively. This is useful for functions that might fail.
```

Comment:
I don't know if this makes sense for this pr, but it might be nice if we added an additional sentence to each data type here like, `The [result library](https://grain-lang.org/docs/stdlib/result) is used for interactions and manipulations`